config/envVarCheck.ts

```diff
@@ -0,0 +1,26 @@
+import { env } from "../src/typeDefinitions/default.types";
+import { loadEnv } from "./config";
+
+validateEnv();
```

Comment:
we are calling function even before declaring it, can we move the function definition up and then call this?
a good general practise